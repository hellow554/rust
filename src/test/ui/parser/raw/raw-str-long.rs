fn main() {
    let a = r##"This //~ ERROR unterminated raw string
    is
    a
    very
    long
    string
    which
    goes
    over
    a
    b
    c
    d
    e
    f
    g
    h
    lines
    "#;
}
