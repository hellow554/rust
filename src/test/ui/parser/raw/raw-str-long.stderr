error: unterminated raw string
  --> $DIR/raw-str-long.rs:2:13
   |
LL |     let a = r##"This
   |             ^^^ unterminated raw string
help: you might have meant to end the raw string here
   |
LL |     "##;
   |     ^^^

error: aborting due to previous error

